plugins/inputs/wireless/wireless_mac.go

```diff
@@ -0,0 +1,78 @@
+//go:build darwin
+// +build darwin
+
+package wireless
+
+import (
+	"errors"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	"os/exec"
+	"runtime"
+	"strconv"
+	"strings"
+	"sync"
+)
+
+// default executable path
+const (
+	OSXCMD = "/System/Library/PrivateFrameworks/Apple80211.framework/Versions/Current/Resources/airport -I"
+)
+
+func (w *Wireless) exe_cmd(cmd string, wg *sync.WaitGroup) ([]byte, error) {
```

Comment:
This method should have a test as well, tests are also run on mac, so it should be possible to run the command.

Reply:
I've removed that method as it was superfluous.